[2023-01-26 17:53:06,623] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2023-01-26 17:53:06,639] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-01T00:00:00+00:00 [queued]>
[2023-01-26 17:53:06,639] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 17:53:06,639] {taskinstance.py:1377} INFO - Starting attempt 9 of 11
[2023-01-26 17:53:06,639] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-01-26 17:53:06,657] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): Leer_Datos_Respuesta_Nasa> on 2023-01-01 00:00:00+00:00
[2023-01-26 17:53:06,663] {standard_task_runner.py:52} INFO - Started process 117 to run task
[2023-01-26 17:53:06,666] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Respuesta_Nasa', 'Leer_Datos_Respuesta_Nasa', 'scheduled__2023-01-01T00:00:00+00:00', '--job-id', '626', '--raw', '--subdir', 'DAGS_FOLDER/00-ETL_spaceX.py', '--cfg-path', '/tmp/tmpfikjrcjy', '--error-file', '/tmp/tmpr8fhqgrh']
[2023-01-26 17:53:06,667] {standard_task_runner.py:80} INFO - Job 626: Subtask Leer_Datos_Respuesta_Nasa
[2023-01-26 17:53:06,745] {task_command.py:371} INFO - Running <TaskInstance: Respuesta_Nasa.Leer_Datos_Respuesta_Nasa scheduled__2023-01-01T00:00:00+00:00 [running]> on host da6de88268cf
[2023-01-26 17:53:06,834] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Respuesta_Nasa
AIRFLOW_CTX_TASK_ID=Leer_Datos_Respuesta_Nasa
AIRFLOW_CTX_EXECUTION_DATE=2023-01-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=9
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-01T00:00:00+00:00
[2023-01-26 17:53:06,835] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-01-26 17:53:06,836] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /tmp && head /tmp/response_20230101.txt']
[2023-01-26 17:53:06,848] {subprocess.py:85} INFO - Output:
[2023-01-26 17:53:06,851] {subprocess.py:92} INFO - ***tmp5ytm60fd
[2023-01-26 17:53:06,851] {subprocess.py:92} INFO - ***tmppo31bdrv
[2023-01-26 17:53:06,851] {subprocess.py:92} INFO - pymp-ysubsmu7
[2023-01-26 17:53:06,851] {subprocess.py:92} INFO - response_20230101.txt
[2023-01-26 17:53:06,852] {subprocess.py:92} INFO - tmpnvc2r2ky
[2023-01-26 17:53:06,852] {subprocess.py:92} INFO - tmpr8fhqgrh
[2023-01-26 17:53:06,852] {subprocess.py:92} INFO - tmpr9m_kfo1
[2023-01-26 17:53:06,852] {subprocess.py:92} INFO - Confirmaci√≥n de la NASA, pueden proceder
[2023-01-26 17:53:06,853] {subprocess.py:96} INFO - Command exited with return code 0
[2023-01-26 17:53:06,887] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=Respuesta_Nasa, task_id=Leer_Datos_Respuesta_Nasa, execution_date=20230101T000000, start_date=20230126T175306, end_date=20230126T175306
[2023-01-26 17:53:06,917] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-01-26 17:53:06,971] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
